lib/floe/workflow.rb

```diff
@@ -60,10 +60,11 @@ def step
       context.states << context.state
 
       @status = current_state.status
-      @output = output if end?
 
       next_state_name = next_state&.name
-      @current_state = next_state_name && @states_by_name[next_state_name]
+      @current_state  = next_state_name && @states_by_name[next_state_name]
```

Comment:
think this is not a change

Reply:
Was aligning the `=` up, but was from an unrelated change I'll pull into a separate PR